pom.xml

```diff
@@ -24,7 +24,7 @@
 		<maven.compiler.release>11</maven.compiler.release>
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>11</maven.compiler.target>
```

Comment:
Notice that I did not modify the release/source/target.

Reply:
Actually, do we need all three of them?

Reply:
Since we're building with Java 9+, the answer is yes. See https://stackoverflow.com/questions/59049980/maven-compiler-release-as-an-replacement-for-source-and-target.

I'll remove these from all poms.

Reply:
Although we do have to add a later version of the Maven compiler plugin explicitly. See https://bugs.eclipse.org/bugs/show_bug.cgi?id=566704.